core/src/main/java/org/arquillian/cube/impl/client/ForceStopDockerContainersShutdownHook.java

```diff
@@ -1,21 +1,35 @@
 package org.arquillian.cube.impl.client;
 
-import java.util.List;
 import org.arquillian.cube.spi.Cube;
 import org.arquillian.cube.spi.CubeRegistry;
 import org.jboss.arquillian.core.api.annotation.Observes;
 import org.jboss.arquillian.test.spi.event.suite.BeforeSuite;
 
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
 public class ForceStopDockerContainersShutdownHook {
 
     public void attachShutDownHookForceStopDockerContainers(@Observes(precedence = 200) BeforeSuite event,
-        final CubeRegistry cubeRegistry) {
+                                                            final CubeRegistry cubeRegistry) {
+
         Runtime.getRuntime().addShutdownHook(new Thread(() -> {
             final List<Cube<?>> cubes = cubeRegistry.getCubes();
-            for (Cube cube : cubes) {
+            for (Cube cube: cubes) {
 
+                try {
                     cube.stop();
-                    cube.destroy();
+                } catch (final Throwable e) {
+                    Logger.getLogger(ForceStopDockerContainersShutdownHook.class.getName()).log(Level.WARNING, "Falied to stop: " + cube.getId(), e);
+                } finally {
+                    try {
+                        cube.destroy();
```

Comment:
@AndyGee Suppose cube failed to stop container, then you can destroy it but with `-f` option, to remove it forcefully. We are not using `-f` option in cube while destroying it

Just curious to know why cube failed to stop container(`vnc` from exception)?

Reply:
Sure, the issue is here that cube starts several containers in a test. If the first container fails to stop for 'any' reason then stop is never called on the following containers (due to the exception).

The secondary issue is that vnc was not started properly (I think that is another cube/drone issue), which is why the call to stop fails. 

This PR ensures that the 'other' containers are stopped and destroyed even if one fails. Throwing an error and stacktrace on the command line is not the expectation, and is not a test failure - Hence the logged warning.